packages/calypso-e2e/src/hooks.ts

```diff
@@ -64,24 +64,26 @@ export const setupHooks = ( callback: ( { page }: { page: Page } ) => void ): vo
 			await page.screenshot( { path: fileName } );
 		}
 		// Close the browser. This needs to be called before trying to access
-		// the video recording
+		// the video recording.
 		await page.close();
-		// Save video
+
 		if ( __CURRENT_TEST_FAILED__ ) {
-			const original = await ( page.video() as Video ).path();
+			// const original = await ( page.video() as Video ).path();
 			const destination = path.join(
 				artifactPath,
 				'screenshots',
 				`${ getTestNameWithTime( testName ) }.webm`
 			);
 			try {
-				await rename( original, destination );
+				// Save the failing test case with a specific name.
+				await page.video()?.saveAs( destination );
 			} catch ( err ) {
-				console.error( 'Failed to rename video of failing test case.' );
+				console.error( `Failed to save video of failing test case.\nStack trace: ${ err }` );
```

Comment:
Tiny nit: but I actually think that string converting an `Error` just prints its `message` - just as a heads up to you that you might not get a stack trace out of this! (which is okay!)

Reply:
Ah, gotcha. In Python, outputting the `err` would have done the trick - will replace it with `console.trace`.